examples/air-purifier-app/air-purifier-common/air-purifier-app.matter

```diff
@@ -146,17 +175,41 @@ cluster AccessControl = 31 {
     fabric_idx fabricIndex = 254;
   }
 
+  fabric_sensitive info event access(read: administer) AccessRestrictionEntryChanged = 2 {
+    fabric_idx fabricIndex = 254;
+  }
+
+  fabric_sensitive info event access(read: administer) FabricRestrictionReviewUpdate = 3 {
+    int64u token = 0;
+    nullable char_string instruction = 1;
+    nullable char_string redirectURL = 2;
+    fabric_idx fabricIndex = 254;
+  }
+
   attribute access(read: administer, write: administer) AccessControlEntryStruct acl[] = 0;
   attribute access(read: administer, write: administer) optional AccessControlExtensionStruct extension[] = 1;
   readonly attribute int16u subjectsPerAccessControlEntry = 2;
   readonly attribute int16u targetsPerAccessControlEntry = 3;
   readonly attribute int16u accessControlEntriesPerFabric = 4;
+  readonly attribute optional CommissioningAccessRestrictionEntryStruct commissioningARL[] = 5;
```

Comment:
my copy of the spec says:

```
ifdef::in-progress,hrap-1[]
| 0x0005  s| ManagedACL                    | <<ref_DataTypeList>>[<<ref_AccessControlEntryStruct>>]
                                                     | desc       |         | desc    | R V F  | MNGD
endif::[]
```

So the attributes are different. please provide information where this PR is basing the config from (likely an in progress spec PR - please provide it in the summary)